### DISCUSSION TITLE
get the current working directory

### DISCUSSION THREAD

**Discussion by gilgamsh** [nushell/nushell#9226] [Q&A]
just like `pwd` in bash. How can i get the current working directory?

**Comment by amtoine** [comment]
you can use the `pwd` command, which i think it not a `bash` command but a system command, e.g. i have it installed in `/usr/bin/` :open_mouth: 

or, if you want to do it in a pure Nushell way, you can use `$env.PWD` :yum:

**Comment by fdncred** [comment]
I have a custom command in my config somewhere that does `def pwd [] { $env.PWD }`
